desktop_server/src/com/mygdx/game/server/network/gameinstance/services/AttackEntityService.java

```diff
@@ -0,0 +1,46 @@
+package com.mygdx.game.server.network.gameinstance.services;
+
+import com.artemis.World;
+import com.mygdx.game.core.ecs.component.CanAttack;
+import com.mygdx.game.core.ecs.component.Coordinates;
+import com.mygdx.game.core.ecs.component.Stats;
+import lombok.extern.java.Log;
+
+import javax.inject.Inject;
+
+@Log
+public class AttackEntityService extends WorldService {
+
+  @Inject
+  public AttackEntityService() {
+    super();
+  }
+
+  public void attackEntity(int attacker, int attacked, World world) {
+    var coordinatesMapper = world.getMapper(Coordinates.class);
```

Comment:
Może inject tak jak teraz robimy?

Reply:
Jak będzie wszystko miało tego gameinstance scope to może trzymać taki stan, więc ok.